ws-nextjs-app/pages/[service]/send/[id]/SubmitButton/index.stories.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import ThemeProvider from '#app/components/ThemeProvider';
+import SubmitButtonComponent from '.';
+
+const Component = () => (
+  <ThemeProvider service="pidgin">
+    <SubmitButtonComponent />
+  </ThemeProvider>
+);
+
+export default {
+  title: 'New Components/SubmitButton',
```

Comment:
Should this component live within the FormFields directory?

```suggestion
title: 'New Components/FormFields/SubmitButton',
```